extensions/wikia/CommunityPage/styles/components/_todo-list-module.scss

```diff
@@ -42,21 +42,21 @@ $module-header-line-height: 28px;
 		clear: both;
 	}
 
-	.community-page-todo-list-module-content {
-		border-style: solid;
+	%community-page-todo-list-module-placeholder {
```

Comment:
> Sorry Konrad, ładne, fancy… no ale przodkowie. :|

Could you separate those classes with coma instead, to make it easier to understand?